### PR TITLE
docs: Update unit-openapi.yaml

### PR DESCRIPTION
**Pull request by Jcahilltorre**
Update openapi for the modules status object

### CODE REVIEW

File: docs/unit-openapi.yaml [hunk 1]
```diff
@@ -6331,6 +6371,11 @@ components:
         applications:
           $ref: "#/components/schemas/statusApplications"
 
+    # /status/modules
+    statusModules:
+      description: "Lists currently loaded language modules."
+      type: object
+
```
Comment (Jcahilltorre) [inline docs/unit-openapi.yaml:6378]: This section is incomplete

File: docs/unit-openapi.yaml [hunk 2]
```diff
@@ -6319,9 +6356,12 @@ components:
 
     # /status
     status:
-      description: "Represents Unit's usage statistics."
+      description: "Represents Unit's loaded language modules and usage statistics."
       type: object
       properties:
+        modules:
+          $ref: "#/components/schemas/statusModules"
+
```
Comment (Jcahilltorre) [inline docs/unit-openapi.yaml:6364]:
```suggestion

```

File: docs/unit-openapi.yaml [hunk 3]
```diff
@@ -3924,13 +3924,38 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/statusModules"
+
+              examples:
+                example1:
+                  $ref: "#/components/examples/statusModules"
+
```
Comment (Jcahilltorre) [inline docs/unit-openapi.yaml:3951]:
```suggestion

```

File: docs/unit-openapi.yaml [hunk 4]
```diff
@@ -3924,13 +3924,37 @@ paths:
...
+            the configuration."
+
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/statusModules"
+
+              examples:
+                example1:
+                  $ref: "#/components/examples/statusModules"
```
Comment (Jcahilltorre) [inline docs/unit-openapi.yaml:3950]:
```suggestion
                  $ref: "#/components/examples/statusModules"

```

File: docs/unit-openapi.yaml [hunk 5]
```diff
@@ -6319,9 +6356,11 @@ components:
 
     # /status
     status:
-      description: "Represents Unit's usage statistics."
+      description: "Represents Unit's loaded language modules and usage statistics."
       type: object
       properties:
+        modules:
+          $ref: "#/components/schemas/statusModules"
```
Comment (Jcahilltorre) [inline docs/unit-openapi.yaml:6363]:
```suggestion
          $ref: "#/components/schemas/statusModules"

```

**Review by ac000 — CHANGES_REQUESTED** [review]
Hi @Jcahilltorre Just a heads up because we want to keep a clean commit history in Unit, without a bunch of unnecessary commits which just adds noise to the commit history.

Taking a quick look at the patches, it seems to me these should be done as two commits as these are two logical changes

1) A first commit that does the URL change
2) A second commit that adds the info for the new modules endpoint

Having the next version of the patches presented like that would help with reviewing!.

It also seems there are some [whitespace issues](https://github.com/nginx/unit/pull/1373/checks)...

Thanks for the patch! I'm still not sure what this is actually used for and if there is anyway to verify its correctness?

**Pull request closed without merging**